google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginAction.java

```diff
@@ -69,15 +65,14 @@ public void update(AnActionEvent e) {
       if (image == null) {
         presentation.setIcon(GoogleLoginIcons.DEFAULT_USER_AVATAR);
```

Comment:
Oh you mean loading over HTTP instead of local assets? We could do that if you want.